jobs/release.Jenkinsfile

```diff
@@ -106,6 +107,21 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
         def gcp_image = ""
         def ostree_prod_refs = [:]
 
+        // arbitrarily selecting the s390x builder; we don't run this
+        // locally because podman wants user namespacing (yes, even just
+        // to push a manifest...)
+        pipeutils.withPodmanRemoteArchBuilder(arch: 's390x') {
+        def session = pipeutils.makeCosaRemoteSession(
+            env: container_env,
+            expiration: "30m",
```

Comment:
any reason for choosing `30m` here? might be nice to make it at least `60m` to be able to investigate failures on the remote builder.